utils/generate_types/exported.json

```diff
@@ -6,5 +6,6 @@
   "BrowserNewContextOptions": "BrowserContextOptions",
   "BrowserNewContextOptionsViewport": "ViewportSize",
   "BrowserNewContextOptionsGeolocation": "Geolocation",
-  "BrowserNewContextOptionsHttpCredentials": "HTTPCredentials"
+  "BrowserNewContextOptionsHttpCredentials": "HTTPCredentials",
+  "PageScreenshotOptions": "ScreenshotOptions",
```

Comment:
I think this is a bit confusing, because `elementHandle.screenshot` does not take these options (it is missing the clip). WDYT?

Reply:
PageScreenshotOptions?

Reply:
👍